mso/resource_mso_schema_site.go

```diff
@@ -106,21 +108,31 @@ func resourceMSOSchemaSiteImport(d *schema.ResourceData, m interface{}) ([]*sche
 		apiSiteId := models.StripQuotes(tempCont.S("siteId").String())
 		apiTemplate := models.StripQuotes(tempCont.S("templateName").String())
 
-		if apiSiteId == stateSiteId {
+		if apiSiteId == stateSiteId && apiTemplate == template {
+			// d.SetId(fmt.Sprintf("%s/sites/%s-%s", schemaId, stateSiteId, template))
 			d.SetId(apiSiteId)
 			d.Set("schema_id", schemaId)
 			d.Set("site_id", apiSiteId)
 			d.Set("template_name", apiTemplate)
+
+			// when undeploy var is provided set it else set default value
+			if len(get_attribute) == 7 {
```

Comment:
Should there also be a check that the 6th attribute is actually `undeploy_on_destroy`?

Reply:
Does not matter, could be any arbitrary string as long as the string after is a boolean parsable string. This is similar to current behaviour of importing commands